### PR TITLE
v6.0.0-alpha.1

### PR DESCRIPTION
**Pull request by m4theushw**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,110 @@
...
 
+## 6.0.0-alpha.1
+<!-- generated comparing v6.0.0-alpha.0..next -->
+_Sep 29, 2022_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Better support for custom overlays (#5808) @cherniavskii
+- 🖨️ Improve print export (#6273) @oliviertassinari
+- 🎁 Support slots on the `DateField` component  (#6048) @flaviendelangle
```
[the remark this answers is not in the document]
Comment (flaviendelangle) [reply CHANGELOG.md:14]: The field are not officially released yet so I would avoid putting them in the highlights :+1: 
The main change is `- [pickers] Do not support unparsed date formats anymore (#6170) @flaviendelangle`

But it should be in the breaking changes (I'll add a comment)

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,110 @@
...
+- [DataGrid] Revert cell/row mode if `processRowUpdate` fails (#6185) @m4theushw
+- [DataGrid] Rework overlays layout (#5808) @cherniavskii
+- [DataGrid] Improve print support (#6273) @oliviertassinari
+- [DataGridPremium] Add missing `themeAugmentation` module (#6270) @cherniavskii
+
+### `@mui/x-date-pickers@v6.0.0-alpha.1` / `@mui/x-date-pickers-pro@v6.0.0-alpha.1`
+
+#### Changes
+
+- [DatePicker] Open in a valid month (#6309) @alexfauquette
```
Comment (flaviendelangle) [inline CHANGELOG.md:80]:
```suggestion
- [DatePicker] Respect `minDate` and `maxDate` when opening a `DatePicker` or `DateTimePicker` (#6309) @alexfauquette
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,110 @@
...
+- [DateTimePicker] Fix validation with `shouldDisableMonth` and `shouldDisableYear` (#6266) @flaviendelangle
+- [TimePicker] Add support for `disablePast` and `disableFuture` validation props (#6226) @LukasTy
+- [CalendarPicker] Prevent getting focus when `autoFocus=false` (#6304) @alexfauquette
+- [DateField] Extend moment adapter to support `expandFormat` and `formatTokenMap` (#6215) @alexfauquette
+- [pickers] Allow to control the selected sections (#6209, #6307) @flaviendelangle
+- [pickers] Do not loose the value of date sections not present in the format in the new field components (#6141) @flaviendelangle
+- [pickers] Do not support unparsed date formats anymore (#6170) @flaviendelangle
+- [pickers] Support slots on the `DateField` component  (#6048) @flaviendelangle
+- [pickers] Try adapter to luxon v3 (#6069) @alexfauquette
```
Comment (flaviendelangle) [inline CHANGELOG.md:89]:
```suggestion
- [pickers] Support Luxon v3 in `AdapterLuxon` (#6069) @alexfauquette
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,110 @@
...
+- [DataGrid] Fix `undefined` value being showed in filter button tooltip text (#6259) @cherniavskii
+- [DataGrid] Fix blank space when changing page with dynamic row height (#6049) @m4theushw
+- [DataGrid] New internal rows structure for v6 (#4927) @flaviendelangle
+- [DataGrid] Revert cell/row mode if `processRowUpdate` fails (#6185) @m4theushw
+- [DataGrid] Rework overlays layout (#5808) @cherniavskii
+- [DataGrid] Improve print support (#6273) @oliviertassinari
+- [DataGridPremium] Add missing `themeAugmentation` module (#6270) @cherniavskii
+
+### `@mui/x-date-pickers@v6.0.0-alpha.1` / `@mui/x-date-pickers-pro@v6.0.0-alpha.1`
+
```
Comment (flaviendelangle) [inline CHANGELOG.md:77]: #### Breaking changes

- The `value` prop of the pickers now expects a parsed value.
  Until now, it was possible to provide any format that your date management library was able to parse For instance, you could pass `value={new Date()}` when using `AdapterDayjs`.

  This brought a lot of confusion so we decided to remove this behavior. The format expected by the `value` prop is now the same as for any other prop holding a date.
  Here is the syntax to initialize a date picker at the current date for each adapter:

  ```tsx
  // Date-fns
  <DatePicker value={new Date()} />

  // Dayjs
  import dayjs from 'dayjs'
  <DatePicker value={dayjs()} />

  // Moment
  import moment from 'moment'
  <DatePicker value={moment()} />

  // Luxon
  import { DateTime } from 'luxon'
  <DatePicker value={DateTime.now()} />
  ```

**Comment by flaviendelangle** [comment]
I merged https://github.com/mui/mui-x/pull/5958 and https://github.com/mui/mui-x/pull/6312

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,137 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.1
+<!-- generated comparing v6.0.0-alpha.0..next -->
```
Comment (alexfauquette) [inline CHANGELOG.md:7]:
```suggestion
```

**Review by alexfauquette — APPROVED** [review]

**Review by flaviendelangle — APPROVED** [review]

**Review by cherniavskii — APPROVED** [review]

**Pull request merged**